docs/user_guide.rst

```diff
@@ -4,6 +4,25 @@ User Guide
 General Information
 *********************
 
+.. dropdown:: Tips for Input
+
+    .. role:: python(code)
+        :language: python
+    
+    Sparcle_QC depends on a properly formatted PDB file. Make sure your PDB follows the standard format. Sparcle_QC does not require occupancy, temperature factor, segment identifier, or formal charge, but it does require all other columns. 
+
+    **CHARMM vs Amber**
+
+    If using CHARMM, add a psf file (downloaded from CHARMM-GUI) to your working directory. Point to the psf file in the Sparcle_QC input with keyword :python:`charmm_psf`. Add the protein PDB's path under the :python:`pdb_file` keyword. Separately add the ligand named "ligand.pdb" to your working directory.
```

Comment:
is this going to be just a section on like pdb prep, should we put somewhere in here more info on like the charmmrst and charmmprm that is needed and maybe like include the link where they can download the tar for all of these (i can't remember if we included it in the repo or not, maybe safer not to)